examples/atari/a3c/README.md

```diff
@@ -0,0 +1,125 @@
+# A3C
+This example trains an Asynchronous Advantage Actor Critic (A3C) agent, from the following paper: [Asynchronous Methods for Deep Reinforcement Learning](https://arxiv.org/abs/1602.01783). 
+
+## Requirements
+
+- atari_py>=0.1.1
+- opencv-python
+
+## Running the Example
+
+```
+python train_a3c.py [options]
+```
+
+### Useful Options
+- `--gpu`. Specifies the GPU. If you do not have a GPU on your machine, run the example with the option `--gpu -1`. E.g. `python train_a3c.py --gpu -1`.
+- `--env`. Specifies the environment. 
+- `--render`. Add this option to render the states in a GUI window.
+- `--seed`. This option specifies the random seed used.
+- `--outdir` This option specifies the output directory to which the results are written.
+
+To view the full list of options, either view the code or run the example with the `--help` option.
+
+## Results
+These results reflect ChainerRL  `v0.6.0`. The ChainerRL score currently consists of a single run. The reported results are compared against the scores from the [Noisy Networks Paper](https://arxiv.org/abs/1706.10295), since the original paper does not report scores for the no-op evaluation protocol.
+
+| Results Summary ||
+| ------------- |:-------------:|
+| Number of seeds | 1 |
+| Number of common domains | 52 |
+| Number of domains where paper scores higher | 25 |
+| Number of domains where ChainerRL scores higher | 24 |
+| Number of ties between paper and ChainerRL | 3 | 
+
+
+
+| Game        | ChainerRL Time           |
```

Comment:
This table is about training time, not scores.

Reply:
| Game        | ChainerRL Score           | Original Reported Scores |
| ------------- |:-------------:|:-------------:|
| AirRaid | 4625.9| N/A|
| Alien | 1397.2| **2027**|
| Amidar | **1110.8**| 904|
| Assault | **5821.6**| 2879|
| Asterix | 6820.7| **6822**|
| Asteroids | 2428.8| **2544**|
| Atlantis | **732425.0**| 422700|
| BankHeist | **1308.9**| 1296|
| BattleZone | 5421.1| **16411**|
| BeamRider | 8493.4| **9214**|
| Berzerk | **1594.2**| 1022|
| Bowling | 31.7| **37**|
| Boxing | **98.1**| 91|
| Breakout | **533.6**| 496|
| Carnival | 5132.9| N/A|
| Centipede | 4849.9| **5350**|
| ChopperCommand | 4881.0| **5285**|
| CrazyClimber | 124400.0| **134783**|
| Defender | N/A| 52917.0|
| DemonAttack | **108832.5**| 37085|
| DoubleDunk | 1.5| **3**|
| Enduro | **0.0**| **0**|
| FishingDerby | **36.3**| -7|
| Freeway | **0.0**| **0**|
| Frostbite | **313.6**| 288|
| Gopher | **8746.5**| 7992|
| Gravitar | 228.0| **379**|
| Hero | **36892.5**| 30791|
| IceHockey | -4.6| **-2**|
| JamesBond | N/A| 509.0|
| Jamesbond | 370.1| N/A|
| JourneyEscape | -871.2| N/A|
| Kangaroo | 115.8| **1166**|
| Krull | **10601.4**| 9422|
| KungFuMaster | **40970.4**| 37422|
| MontezumaRevenge | 1.9| **14**|
| MsPacman | **2498.0**| 2436|
| NameThisGame | 6597.0| **7168**|
| Phoenix | **42654.5**| 9476|
| Pitfall | -10.8| N/A|
| Pitfall! | N/A| 0.0|
| Pong | **20.9**| 7|
| Pooyan | 4067.9| N/A|
| PrivateEye | 376.1| **3781**|
| Qbert | 15610.6| **18586**|
| Riverraid | 13223.3| N/A|
| RoadRunner | 39897.8| **45315**|
| Robotank | 2.9| **6**|
| Seaquest | **1786.5**| 1744|
| Skiing | -16090.5| **-12972**|
| Solaris | 3157.8| **12380**|
| SpaceInvaders | **1630.6**| 1034|
| StarGunner | **57943.2**| 49156|
| Surround | N/A| -8.0|
| Tennis | **-0.3**| -6|
| TimePilot | 3850.6| **10294**|
| Tutankham | **331.4**| 213|
| UpNDown | 17952.0| **89067**|
| Venture | **0.0**| **0**|
| VideoPinball | **407331.2**| 229402|
| WizardOfWor | 2800.0| **8953**|
| YarsRevenge | **25175.5**| 21596|
| Zaxxon | 80.7| **16544**|

Here are the real results. The "Results Summary" remains the same. Once the PR is merged, do you think we should add a caveat saying "this example was evaluated every million timesteps instead of every 250K"?